platform/mbed_error.h

```diff
@@ -265,7 +266,7 @@ typedef enum _mbed_module_type {
     MBED_MODULE_KERNEL,
     MBED_MODULE_NETWORK_STACK,
     MBED_MODULE_HAL,
-    MBED_MODULE_NETWORK_STACKMODULE_MEMORY_SUBSYSTEM,
+    MBED_MODULE_MEMORY_SUBSYSTEM,
```

Comment:
Changing the actual enum name could be breaking for someone already using it.
Please do not update this in docs